src/linter/reporters/console.mjs

```diff
@@ -21,10 +21,10 @@ export default issue => {
     ? ` (${issue.location.position.start.line}:${issue.location.position.end.line})`
     : '';
 
-  process.stdout.write(
+  console[issue.level](
```

Comment:
Can we make sure this function exists before calling it?

Reply:
We know the function exists, the level is either "info", "warn", or "error".